<ui version="4.0" >
 <class>CSSCustomDialog</class>
 <widget class="QWidget" name="CSSCustomDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>526</height>
   </rect>
  </property>
  <property name="whatsThis" >
   <string>&lt;b>Stylesheets&lt;/b>&lt;p>See http://www.w3.org/Style/CSS for further information on cascading style sheets.&lt;/p></string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="3" column="0" >
      <widget class="QGroupBox" name="GroupBox7" >
       <property name="whatsThis" >
        <string>&lt;b>Font family&lt;/b>&lt;p>A font family is a group of fonts that resemble one another, with family members that are e.g. bold, italic, or any number of the above.&lt;/p></string>
       </property>
       <property name="title" >
        <string>Font Family</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="txtBaseFontFamily" >
          <property name="text" >
           <string>Base family:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="KFontComboBox" name="fontFamily" >
          <property name="whatsThis" >
           <string>&lt;p>This is the currently selected font family&lt;/p></string>
          </property>
          <item>
           <property name="text" >
            <string>Sans Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Monospace</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Charter</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Century Schoolbook L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Courier 10 Pitch</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Delphine</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Dingbats</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeMono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSerif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Mono L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Roman No9 L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Sans L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>OpenSymbol</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Standard Symbols L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Steve</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Terminus</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Bookman L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Chancery L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Gothic L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Palladio L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Sans Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Monospace</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Charter</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Century Schoolbook L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Courier 10 Pitch</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Delphine</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Dingbats</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeMono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSerif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Mono L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Roman No9 L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Sans L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>OpenSymbol</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Standard Symbols L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Steve</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Terminus</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Bookman L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Chancery L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Gothic L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Palladio L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Sans Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Monospace</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Charter</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bitstream Vera Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Century Schoolbook L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Courier 10 Pitch</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Sans Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>DejaVu Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Delphine</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Dingbats</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeMono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>FreeSerif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Mono</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Sans</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Liberation Serif</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Mono L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Roman No9 L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Nimbus Sans L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>OpenSymbol</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Standard Symbols L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Steve</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Terminus</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Bookman L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Chancery L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Gothic L</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>URW Palladio L</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QCheckBox" name="sameFamily" >
          <property name="whatsThis" >
           <string>&lt;b>Use same family for all text&lt;/b>&lt;p>Select this option to override custom fonts everywhere in favor of the base font.&lt;/p></string>
          </property>
          <property name="text" >
           <string>Use same family for all text</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QGroupBox" name="GroupBox3" >
       <property name="title" >
        <string>Font Size</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="txtFontsize" >
          <property name="text" >
           <string>Base font si&amp;ze:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>basefontsize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="basefontsize" >
          <property name="editable" >
           <bool>true</bool>
          </property>
          <item>
           <property name="text" >
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>48</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>64</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QCheckBox" name="dontScale" >
          <property name="whatsThis" >
           <string>&lt;b>Use same size for all elements&lt;/b>&lt;p>Select this option to override custom font sizes in favor of the base font size. All fonts will be displayed in the same size.&lt;/p></string>
          </property>
          <property name="text" >
           <string>&amp;Use same size for all elements</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QGroupBox" name="ButtonGroup2" >
       <property name="title" >
        <string>Colors</string>
       </property>
       <layout class="QGridLayout" >
        <item row="3" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="5" >
         <widget class="QRadioButton" name="blackOnWhite" >
          <property name="whatsThis" >
           <string>&lt;b>Black on White&lt;/b>&lt;p>This is what you normally see.&lt;/p></string>
          </property>
          <property name="text" >
           <string>&amp;Black on white</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5" >
         <widget class="QRadioButton" name="whiteOnBlack" >
          <property name="whatsThis" >
           <string>&lt;b>White on Black&lt;/b>&lt;p>This is your classic inverse color scheme.&lt;/p></string>
          </property>
          <property name="text" >
           <string>&amp;White on black</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5" >
         <widget class="QRadioButton" name="customColor" >
          <property name="whatsThis" >
           <string>&lt;b>Custom&lt;/b>&lt;p>Select this option to define a custom color for the default font.&lt;/p></string>
          </property>
          <property name="text" >
           <string>Cus&amp;tom</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="txtBackground" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;b>Background&lt;/b>&lt;p>This background color is the one displayed behind the text by default. A background image will override this.&lt;/p></string>
          </property>
          <property name="text" >
           <string>Bac&amp;kground:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>backgroundColorButton</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="KColorButton" name="backgroundColorButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;b>Background&lt;/b>&lt;p>Behind this door lays the ability to choose a custom default background.&lt;/p></string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="5" >
         <widget class="QCheckBox" name="sameColor" >
          <property name="whatsThis" >
           <string>&lt;b>Use same color for all text&lt;/b>&lt;p>Select this option to apply your chosen color to the default font as well as any custom fonts as specified in a stylesheet.&lt;/p></string>
          </property>
          <property name="text" >
           <string>Use same color for all text</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QLabel" name="txtForeground" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;b>Foreground color&lt;/b>&lt;p>The foreground color is the color that the text is drawn in.&lt;/p></string>
          </property>
          <property name="text" >
           <string>&amp;Foreground:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>foregroundColorButton</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="4" >
         <widget class="KColorButton" name="foregroundColorButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;b>Foreground color&lt;/b>&lt;p>The foreground color is the color that the text is drawn in.&lt;/p></string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QGroupBox" name="GroupBox3_2" >
       <property name="whatsThis" >
        <string>&lt;b>Images&lt;/b></string>
       </property>
       <property name="title" >
        <string>Images</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="hideImages" >
          <property name="whatsThis" >
           <string>&lt;b>Suppress images&lt;/b>&lt;p>Selecting this will prevent Konqueror from loading images.&lt;/p></string>
          </property>
          <property name="text" >
           <string>&amp;Suppress images</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="hideBackground" >
          <property name="whatsThis" >
           <string>&lt;b>Suppress background images&lt;/b>&lt;p>Selecting this option will prevent Konqueror from loading background images.&lt;/p></string>
          </property>
          <property name="text" >
           <string>Suppress background images</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0" colspan="2" >
      <widget class="QGroupBox" name="previewBox" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="title" >
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" />
   </item>
   <item>
    <layout class="QHBoxLayout" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontComboBox</class>
   <extends>KComboBox</extends>
   <header>kfontcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sameFamily</tabstop>
  <tabstop>basefontsize</tabstop>
  <tabstop>dontScale</tabstop>
  <tabstop>blackOnWhite</tabstop>
  <tabstop>backgroundColorButton</tabstop>
  <tabstop>sameColor</tabstop>
  <tabstop>hideImages</tabstop>
  <tabstop>hideBackground</tabstop>
 </tabstops>
 <includes>
  <include location="global" >kcolorbutton.h</include>
  <include location="local" >kcolorbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>customColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>backgroundColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>339</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel" >
     <x>420</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>foregroundColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>318</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>416</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtBackground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>377</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel" >
     <x>355</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtForeground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>390</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>368</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
